### DISCUSSION TITLE
What is useState() in React?

### DISCUSSION THREAD

**Discussion by Akash2377** [Akash2377/Akash2377_portfolio#5] [Q&A] [answered]
What is useState() in React?

**Comment by Manu2807** [comment, marked as answer]
UseState is hook in the react use to manage the state in the react application

**Comment by Pakash2211** [comment]
The useState () is a Hook that allows you to have state variables in functional components. React has two types of components, one is class components which are ES6 classes that extend from React and the other is functional components.

**Comment by abhishekpandey108** [comment]
It is a React hook , we use this hook to maintain the state.
